drivers/hmis_external_apis/README.md

```diff
@@ -0,0 +1,3 @@
+## HmisExternalApis README
+
+This README file should be used to explain the functionality of the driver.
```

Comment:
Can you add a brief overview of what this driver does here?

Reply:
I was planning to do that after I know more about what it's supposed to do.  I'm a little blocked on specs for this branch.